scripts/002-gcc-newlib-PPU.sh

```diff
@@ -11,8 +13,8 @@ if [ ! -d ${GCC} ]; then
   if [ ! -f ${NEWLIB}.tar.gz ]; then wget --continue https://sourceware.org/pub/newlib/${NEWLIB}.tar.gz; fi
 
   ## Unpack the source code.
-  rm -Rf ${GCC} && tar xfvJ ${GCC}.tar.xz
-  rm -Rf ${NEWLIB} && tar xfvz ${NEWLIB}.tar.gz
+  rm -Rf ${GCC} && tar -xvJf ${GCC}.tar.xz
+  rm -Rf ${NEWLIB} && tar -xvf ${NEWLIB}.tar.gz
```

Comment:
Again, no dash.
You can remove the `v` if you want; the printing of the filenames can actually take some time on a slow system and it doesn't really serve any purpose.